test/capture.js

```diff
@@ -1,3 +1,21 @@
+define([ 'intern!tdd',
+        'intern/chai!expect',
+        '../pointerevents',
+        'src/boot',
```

Comment:
I've changed all the test files to use 'src/PointerEvent' now, except for dispatcher.js, which throws "ReferenceError: PointerEventsPolyfill is not defined" unless I include references to the other src files.